Scripting/BindingObject.cs

```diff
@@ -5,6 +5,8 @@ class BindingObject
 	public Bindings.TYPE Type = Bindings.TYPE.UNSET;
 	public Bindings.DIRECTION AxisDirection; //Only used if Type is AXIS
 
+	public bool IsZero = true;
```

Comment:
This should be something like `JoyWasInDeadzone` as that is how it is used
The current name is misleading